app/src/main/java/com/example/mycalculator/mvp/CalculatorPresenterImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.example.mycalculator.mvp;
+
+public class CalculatorPresenterImpl implements CalculatorContract.CalculatorPresenterContract {
```

Comment:
you can change the name of this class and call it CalculatorPresenter

Reply:
Done. Thx!